compiler/damlc/daml-stdlib-src/DA/Text.daml

```diff
@@ -44,6 +46,9 @@ module DA.Text
   , DA.Text.isAlpha
   , DA.Text.isAlphaNum
   , DA.Text.parseInt
+#ifdef DAML_NUMERIC
+  , DA.Text.parseNumeric
+#endif
   , DA.Text.parseDecimal
```

Comment:
Should we deprecate `parseDecimal` (not necessarily now but at some point).

Reply:
At some point, I think so